src/components/app_catalog/AppList/AppListInner.js

```diff
@@ -1,5 +1,5 @@
+import { memoize } from 'lodash';
```

Comment:
`React.memo` is used to optimize the number of updates on React functional components (similar to how `PureComponent` does it for class components).

The reason for memoizing this is to prevent having to filter and parse the whole list every time we re-render the component. This way we avoid a whole lot of CPU usage.

Reply:
Ok, then if you have to use it, I think it would be best to use `Underscore` cause this is the library we are using for these kinf of things.

Reply:
Done 👍